src/App.css

```diff
@@ -33,7 +33,8 @@ span.empty {
 
```

Comment:
This is a generated file, you should have a "build" directory or something similarly named so you don't pollute your source directory with generated files.